tests/libnet/netattachdef.go

```diff
@@ -53,6 +53,15 @@ func NewBridgeNetAttachDef(name, bridgeName string, opts ...pluginConfOption) *n
 	return NewNetAttachDef(name, NewNetConfig(name, pluginConfig))
 }
 
+func NewSriovNetAttachDef(name string, vlanID int, opts ...pluginConfOption) *nadv1.NetworkAttachmentDefinition {
```

Comment:
The SR-IOV CNI considers the [vlan](https://github.com/k8snetworkplumbingwg/sriov-cni/blob/4e22051a92a0f8dedfbe9c13e5e27a492225bb82/docs/configuration-reference.md?plain=1#L11) option as optional. Please consider having it as an option instead of a mandatory field.